libs/ui/src/env.d.ts

```diff
@@ -1,7 +1,6 @@
 declare namespace NodeJS {
   export interface ProcessEnv {
-    readonly NODE_ENV: 'development' | 'production' | 'test';
-    readonly REACT_APP_VX_DEV?: string;
-    readonly REACT_APP_VX_DISABLE_CARD_READER_CHECK?: string;
```

Comment:
did you mean to remove these?

Reply:
Yes, per this slack discussion: https://votingworks.slack.com/archives/CEL6D3GAD/p1665608644655979. They're only necessary when referenced explicitly, as in the tests now.

Reply:
Made these `readonly` again.